examples/src/omezarr/omezarr-viewer.tsx

```diff
@@ -44,30 +50,96 @@ export function OmezarrViewer({
     const canvas = useRef<HTMLCanvasElement>(null);
     const server = useContext(renderServerContext);
     const renderer = useRef<ReturnType<typeof buildAsyncOmezarrRenderer>>();
+    const imgRenderer = useRef<ReturnType<typeof buildImageRenderer>>();
+    const stash = useRef<StashedView>();
 
     // setup renderer and delete it when component goes away
     useEffect(() => {
         const c = canvas?.current;
         if (server?.regl && omezarr) {
             const numChannels = omezarr.colorChannels.length || 3;
-            renderer.current = buildAsyncOmezarrRenderer(server.regl, defaultDecoder, { numChannels });
+            renderer.current = buildAsyncOmezarrRenderer(server.regl, multithreadedDecoder, {
+                numChannels,
+                queueOptions: { maximumInflightAsyncTasks: 1 },
+            });
+            imgRenderer.current = buildImageRenderer(server.regl);
         }
         return () => {
             if (c) {
                 server?.destroyClient(c);
             }
         };
     }, [server, omezarr]);
+    useEffect(() => {
+        // set up the stash:
+        if (server?.regl) {
+            stash.current = {
```

Comment:
here's the smoke-and-mirrors performance improvement: we keep a copy of the rendered screen in "stash".